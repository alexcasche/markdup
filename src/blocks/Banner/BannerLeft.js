import styled from 'styled-components'

const BannerLeft = styled.div`
  flex-basis: 50%;
  padding-right: 10px;
`

export default BannerLeft
