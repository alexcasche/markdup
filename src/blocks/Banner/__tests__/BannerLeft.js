import BannerLeft from '../BannerLeft'

describe('BannerLeft Element', () => {
  it('renders correctly', () => {
    const tree = render(<BannerLeft />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
