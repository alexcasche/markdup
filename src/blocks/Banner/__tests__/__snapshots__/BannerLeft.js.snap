// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerLeft Element renders correctly 1`] = `
.c0 {
  -webkit-flex-basis: 50%;
  -ms-flex-preferred-size: 50%;
  flex-basis: 50%;
  padding-right: 10px;
}

<div
  className="c0"
/>
`;
